### ISSUE TITLE
App loads in white page

### ISSUE THREAD

**Issue by marcellodesales** [atc-/docker-registry-web#115]
Not sure why, but I just pulled a fresh version of the image... and it returns empty... Container is running...

```
[root@pppdc9prd3ga mdesales]# docker ps
CONTAINER ID        IMAGE                             COMMAND                CREATED             STATUS              PORTS                  NAMES
ea780f0bb8c0        atcol/docker-registry-ui:latest   /bin/sh -c /usr/shar   10 minutes ago      Up 10 minutes       0.0.0.0:80->8080/tcp   federated-registry
```

Returning a 404 empty page...

```
mdesales@SDGl121456EB ~ (mac) $ wget http://registry.docker.corp.intuit.net
--2014-11-15 02:59:21--  http://registry.docker.corp.intuit.net/
Resolving registry.docker.corp.intuit.net... 10.132.63.191
Connecting to registry.docker.corp.intuit.net|10.132.63.191|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2014-11-15 02:59:22 ERROR 404: Not Found.
```

**Comment by atc-** [comment]
Resolved in 0.96.1. Releasing shortly...

**Comment by mingliu** [comment]
I got this exception when my docker registry ui got restarted by docker daemon restart in a host reboot, since I set "--restart always" to the container. However, tomcat started but registry ui failed by following exception. I confirmed I have used your fix version of "FILE_LOCK=SOCKET", but seems that was another kind of exception as "locked by another computer", not "by another process", and gave its IP as "172.17.0.2". I suspect it was the container internal IP before restart, and after restart container internal IP has been changed, so the lock failed?
Do you have any advice how could I fix this? Since this is production environment, I have no permission to check inside container. I think one fix is to request our system administrator to remove the container, and recreate one? I have not H2 database mounted to host, I don't think this related?

2015-10-25 04:02:42,521 [localhost-startStop-1] ERROR pool.ConnectionPool  - Unable to create initial connections of pool.
org.h2.jdbc.JdbcSQLException: Database may be already in use: "Locked by another computer: 172.17.0.2". Possible solutions: close all other connection(s); use the server mode [9
0020-176]
        at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
        at org.h2.message.DbException.get(DbException.java:178)
        at org.h2.message.DbException.get(DbException.java:154)
        at org.h2.store.FileLock.getExceptionAlreadyInUse(FileLock.java:462)
